src/reducers/index.js

```diff
@@ -12,9 +12,13 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      count: initialState.count++
```

Comment:
You need to spread state or you will have error where when you try to increment, it won't do anything until you press the button a second time. It also will stay at 0 when you press the async button unless you press twice.

```
      return{
        ...state,
        state: state.count++
      };
```

Reply:
oh I had this. Guess I deleted before I committed, Will update!